app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&.` giải thích code này nhé

Reply:
&. là toán tử điều hướng an toàn, theo như em hiểu thì dòng lệnh thứ 6 có ý nghĩa là kiểm tra user có nil hay không nếu có thì trả về nil, nếu không thì gọi hàm authenticate của user